examples/default/ios/InstabugTests/InstabugSampleTests.m

```diff
@@ -347,8 +346,7 @@ - (void)testNetworkLogIOS {
                            errorCode:errorCode
                            startTime:startTime
                             duration:duration
-                        gqlQueryName:gqlQueryName
-                  serverErrorMessage:serverErrorMessage];
```

Comment:
Why are you removing serverErrorMessage?

Reply:
There has been an update in the IOS native API